src/config.h.in

```diff
@@ -2,6 +2,7 @@
 #define SOLVESPACE_CONFIG_H
 
 #define PACKAGE_VERSION "@PROJECT_VERSION@~@solvespace_GIT_HASH@"
+#define GIT_HASH_URL "https://github.com/solvespace/solvespace/commits/@solvespace_GIT_HASH@"
```

Comment:
```suggestion
#define GIT_HASH_URL "https://github.com/solvespace/solvespace/commit/@solvespace_GIT_HASH@"
```

Maybe use the commit page instead of the commits overview?